### PR TITLE
Try/global styles support

### PR DESCRIPTION
**Pull request by nosolosw**
Work In Progress - see an alternative at https://github.com/WordPress/theme-experiments/pull/22

This is an experiment to try global styles on the Parisienne theme.

- Check out this pull request locally.
- Follow instructions at https://github.com/WordPress/gutenberg/pull/20047

### CODE REVIEW

File: parisienne/experimental-theme.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"color": {
+		"foreground": "#141414",
+		"background": "#F3EBE8",
+		"button-text": "white",
+		"button": "#817359",
```
Comment (youknowriad) [inline parisienne/experimental-theme.json]: Why do we have a "button" and "button-text" variables, these seem too specific to me. Why . not just use "primary" for buttons and "light-text" or something for the button text (same for nav).

I think we should try to limit the number of variables as much as possible and compute the others.

Comment (nosolosw) [reply parisienne/experimental-theme.json]: Those are good thoughts that I also agree with!

The first step was just extracting what Parisienne had and see how that looked. The next iteration for this PR should consolidate them into something general (for what we need to come up with a set of variables and its derivatives).

**Comment by nosolosw** [comment]
Note that this is not ready for review! At the moment it's sort of a playground to demo Global Styles with `edit-site`. If anyone would add the "In Progress" label I'd be grateful (I don't have permissions to do it myself).

File: parisienne/experimental-theme.json (hunk 1 above)
Comment (kjellr) [reply parisienne/experimental-theme.json]: I do agree that these seem overly specific for now. Feel free to just assign primary/background colors here. 

> I think we should try to limit the number of variables as much as possible and compute the others.

I agree that this makes sense for the defaults, but I do think it'll be important to make sure theme authors (and users) can override these computed values with custom ones on a theme + global styles level.

**Comment by nosolosw** [comment]
Closing this in favor of https://github.com/WordPress/theme-experiments/pull/22

**Pull request closed without merging**